make.xsh

```diff
@@ -133,30 +133,65 @@ def start_backend_prod():
     __start_backend(env_file=PROD_ENV_FILE)
 
 @makefile.target(tag="run services locally")
-def backup_database_dev():
+def restore_database():
     """
-    Use the "database-backup" service in the "docker-compose.yml" file to backup
-    all of the tables in the instace of Postgres running locally.
+    Use the "database-backup" service in the "docker-compose.yml" file drop, recreate,
+    and restore all of the tables from S3.
     """
     export_dot_env_vars(env_file=DEV_ENV_FILE)
+    export_rootski_profile_aws_creds()
     $POSTGRES_HOST = get_localhost()
-    docker network prune --force
-    docker-compose run database-backup backup
+    # Checks for a running Postgres container and restores the database if it is running.
+    docker ps | grep postgres && \
+        docker-compose run database-backup restore-from-most-recent || \
+        echo "There is not a Postgres container currently running. Run `make start-database-stack` to start the database."
 
 @makefile.target(tag="run services locally")
-def restore_database():
+def backup_database():
```

Comment:
Nice! I imagine this was the hardest one to write. When you run `make start-database-stack`, does this one work? No issues with `docker-compose` and `docker stack deploy` not working well together?

Reply:
Same question with `make restore-database`

Reply:
These now just find the database-backup container that is running the backup-database-on-interval command in the docker swarm and execute the restore-database-from-most-recent and backup-database on that container instead of starting up a second database-backup container